src/main/java/core/basesyntax/strategy/impl/Purchase.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.strategy.OperationHandler;
+
+public class Purchase implements OperationHandler {
+    @Override
+    public void handleFruitOperation(String fruit, Integer quantity) {
```

Comment:
Maybe integer nice, but I think int better, because you can't get NPE in primitive types